src/components/Card.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react'
+import CardModal from '../components/CardModal'
+import '../styles/Card.css'
+
+type CardProps = {
+  alt?: string
+  media_type?: string
+  title?: string
+  date?: string
+  src: string
+  highDefSrc?: string
+  explanation?: string
+  copyright: string
+  likedImage: boolean
+  innerRef?: (node: HTMLDivElement) => void
+  likeAction?: () => void
+}
+
+function Card({
+  alt,
+  media_type,
+  title,
+  date,
+  src,
+  highDefSrc,
+  explanation,
+  copyright,
+  likedImage,
+  innerRef,
+  likeAction,
+}: CardProps) {
+  const [modalStatus, setModalStatus] = useState(false)
+
+  const dayInLetters = date ? `${new Date(date).toDateString().slice(0, 3)},` : 'No Date'
+  const dayInNumbersAndYear = date ? new Date(date).toDateString().slice(7) : 'No Date'
+  let media = <img src={src} alt={alt} onClick={() => setModalStatus(true)}></img>
+
+  if (media_type === 'video')
+    media = <iframe src={src} title={title ? title : alt} onClick={() => setModalStatus(true)}></iframe>
```

Comment:
Sorry, this is a little nitpicky. In certain cases, you can write shorthand versions of ternaries using the OR operator (if the left side is falsy, take the value on the right). I like to write these when I'm sure I can rely on a single default value to fall back on. 
```suggestion
    media = <iframe src={src} title={title || alt} onClick={() => setModalStatus(true)}></iframe>
```